pkg/http/subscriber.go

```diff
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/v5"
```

Comment:
Hi can you also update the import in pkg/http/sse_test.go so that we get rid of the v4 import entirely?

Reply:
fixed

Reply:
@laouji would you mind to do a CR? thanks